main.js

```diff
@@ -0,0 +1,319 @@
+// 1
+
+const thingsInHome = ["mint", "basil", "cactus", "table", "wooden spoon", "bread"];
+const thingsInGarden = ["apple", "trees", "stairs", "plum", "wooden bench"];
+
+const madeOutOfWood = [thingsInHome[3], thingsInHome[4], thingsInGarden[1], thingsInGarden[4]];
+const edibles = [thingsInHome[0], thingsInHome[1], thingsInHome[5], thingsInGarden[0], thingsInGarden[3]];
+
+console.log(
+    `made out of wood: ${JSON.stringify(madeOutOfWood)}\n edibles: ${JSON.stringify(edibles)}`
+);
+
+// 2
+
+function getSmallestNumber(numbersArray) {
+    let smallestNumber = numbersArray[0];
+
+    for (i = 1; i < numbersArray.length; i++) {
+        if (numbersArray[i] < smallestNumber) {
+            return smallestNumber = numbersArray[i];
+        }
+    }
+    return smallestNumber;
+}
+
+function getSmallestNumber(numbersArray) {
+    return numbersArray.sort(
+        function (firstNumber, secondNumber) {
+            return firstNumber - secondNumber;
+        }
+    );
+}
+
+console.log(getSmallestNumber([2, -5, 10, 1, 4])); // -5
+console.log(getSmallestNumber([200, 25, 4, 123, 87])); // 4
+
+// 3
+
+function getSquaredNumbers(numbersArray) {
+    let squaredNumbers = [];
+
+    for (let i = 0; i < numbersArray.length; i++) {
+        squaredNumbers.push(numbersArray[i] * numbersArray[i]);
```

Comment:
You are using `numbersArray[i]` twice. It would make sense to declare a variable with a descriptive name.